rasa/shared/importers/importer.py

```diff
@@ -424,6 +428,16 @@ def get_flows(self) -> FlowsList:
 
         return flows.merge(FlowsList(missing_default_flows))
 
+    @rasa.shared.utils.common.cached_method
+    def get_flows(self) -> FlowsList:
+        flows = self._importer.get_flows()
+
+        if flows.is_empty():
+            # if there are no flows, we don't need to add the default flows either
+            return flows
+
+        return self.merge_with_default_flows(flows)
```

Comment:
Good thinking to keep DM1 bots clean, do we do the same for the domain?

Reply:
yes